src/lexer/tokens.js

```diff
@@ -87,10 +87,10 @@ const Tokens = {
 
   Comment:              '//',                      // a run of text that we ignore
 
-  Identifier:           /[a-zA-Z0-9_:.]+/,         // a single word (used for functions)
+  Identifier:           /[ a-zA-Z0-9_:.]+/,         // words (used for functions)
```

Comment:
Unfortunately this is not an acceptable change. an identifier is equivalent to a variable name in a programming language, so it shouldn't have spaces in it. The fact that this didn't break any tests means the tests are lacking lol, whoops.
We'll need to find another way

Reply:
ah ok, I thought that its a bit hacky

Reply:
Ok you got me there xD I have no idea how to remove the ugly hack, without breaking the parser tests. I get this error:

  1) Dialogue Returns a command to the user:
     Error: Parse error on line 2: Unexpected 'Text'
      at Parser.parseError (eval at createParser (node_modules\jison\lib\jison.js:1327:13), <anonymous>:156:21)
      at Parser.parse (eval at createParser (node_modules\jison\lib\jison.js:1327:13), <anonymous>:223:22)
      at Runner.run (src\runner.js:84:43)
      at run.next (<anonymous>)
      at Context.it (tests\test_runner.js:397:21)

When I take it out

Reply:
If I add () to the identifier token, that breaks a whole bunch of other tests.. 
Whatever angle I try with it, it breaks

Reply:
It works if I create a new token for commandIdentifier , but we still need to keep the ugly hack.

Reply:
Ok no need to use the ugly hack any more and also fixed this without changing the Identifier's regex